### ISSUE TITLE
Can the training data be provided?

### ISSUE THREAD

**Issue by yunh-w** [ZJU-REAL/GUI-RCPO#1]
Could you kindly share if the training data is available for reference?

**Comment by DIONG666** [comment]
Thank you for your interest in our work. We use the ScreenSpot-v2 dataset as the training data to conduct test-time reinforcement learning, without leveraging the ground-truth labels. To reproduce our results, please organize the ScreenSpot-v2 data into the JSONL format following the structure illustrated in our provided example. We hope this helps address your issue.

**Comment by yunh-w** [comment]
@DIONG666 What does 'without leveraging the ground-truth labels' mean? Also, could you kindly provide this JSONL file if possible? Thank you very much!
